src/components/Users/UsersUpdate.jsx

```diff
@@ -0,0 +1,128 @@
+/* eslint react/prop-types: 0 */
+import React, { Component } from 'react';
+import { Container, Row, Col, Form, Button } from 'react-bootstrap';
+import API from '../../apis/API';
+
+class UsersUpdate extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      userID: null,
+      userLevelID: null,
+      firstName: null,
+      lastName: null,
+      successMessage: null,
+      errorMessage: null,
+    };
+
+    this.getUser = this.getUser.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    const { location } = this.props;
+
+    this.setState({ userID: location.pathname.slice(11) }, () =>
```

Comment:
Why `setState` on a `prop`? We shouldn't expect it to change, we can refer to props in the `getUser` call

Also, `this.props.match.params.id` is an easy way to pull the prop

Reply:
I fixed the handling of userID.

I was not sure how we wanted to pass the id but I went with "history.push(`/user-edit/${selected}`);" Instead of passing id in params to the componenet.